docs/admin/kubelet.md

```diff
@@ -88,6 +88,7 @@ kubelet
       --event-burst=10: Maximum size of a bursty event records, temporarily allows event records to burst to this number, while still not exceeding event-qps. Only used if --event-qps > 0
       --event-qps=5: If > 0, limit event creations per second to this value. If 0, unlimited.
       --eviction-hard="": A set of eviction thresholds (e.g. memory.available<1Gi) that if met would trigger a pod eviction.
+      --eviction-pressure-transition-frequency=5m0s: Duration for which the kubelet has to wait before transitioning out of an eviction pressure condition.
```

Comment:
I tried to be consistent with 

```
--outofdisk-transition-frequency=5m0s: Duration for which the kubelet has to wait before transitioning out of out-of-disk node condition status. Default: 5m0s
```

but I tend to agree that period is better.